Dockerfile

```diff
@@ -1,15 +1,20 @@
-FROM node:20.12.2
+FROM node:20.12.2-slim
+
+RUN apt-get update || : && apt-get install -y \
```

Comment:
`A || : B` seems obscure; is it possible just to write `A; B`?

Reply:
Also I'm not sure I've previously seen `apt-get update` failures handled in Dockerfiles before; is there a reason specifically to use it here?

Reply:
I copied this from an example on switching to a slim image. But can simplify!